docker/build.sh

```diff
@@ -31,7 +31,7 @@ IMAGES=(
 )
 
 function docker_push {
```

Comment:
Remove this commented out line please.

Reply:
This is the line that pushes latest, forgot to uncomment this guy to restore the file to its original state.